msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -1066,6 +1066,23 @@ private static PublicClientApplicationConfiguration loadConfiguration(InputStrea
                 throw new IllegalArgumentException("Unable to open provided configuration file.", e);
             }
         }
+        finally {
+            try {
+                configStream.close();
+            } catch (IOException e) {
+                if (isDefaultConfiguration) {
+                    com.microsoft.identity.common.internal.logging.Logger.verbose(
```

Comment:
Can you log this at `Logger.warn` instead of `Logger.verbose`?

Reply:
👍